### ISSUE TITLE
Unable to install googleCloudStorageR

### ISSUE THREAD

**Issue by pehkawn** [cloudyr/googleCloudStorageR#116]
Hi,
I'm currently trying to install the stable version of googleCloudStorageR, but R seems unable to find the package.
Typing `install.packages("googleCloudStorageR")` returns the following:

```
Installing package into ‘/home/nord06019185/R/x86_64-pc-linux-gnu-library/3.6’
(as ‘lib’ is unspecified)
also installing the dependencies ‘fs’, ‘sys’, ‘gargle’, ‘memoise’, ‘askpass’, ‘curl’, ‘googleAuthR’, ‘httr’, ‘openssl’, ‘zip’
```
Then followed by trying a series of URLs, followed by the following error message for each of the listed dependencies and the main package:
```
Error in library(googleCloudStorageR) : 
  there is no package called ‘googleCloudStorageR’
Execution halted
Warning in install.packages :
  installation of package ‘<package name>’ had non-zero exit status
```

What could be the issue?

**Comment by pehkawn** [comment]
Thanks for your reply. 
After installing a few Linux-specific tools, I was now able to install all dependencies and the main package. 
I also tried installing `googleLangugeR`, which also returned similar error message. However, in that case then installation worked when I closed the project I was working on. Seemingly the defined authentication variables in `.Renviron` seemed to interfere with the installation process. Don't know if this has anything to do with this issue.
